deps/v8/gypfiles/v8.gyp

```diff
@@ -418,12 +418,13 @@
           'toolsets': ['host', 'target'],
           'dependencies': [
             'mksnapshot#host',
-            'js2c#host',
+            'js2c_extras#host',  # TODO(refaack) get rid of this
```

Comment:
Typo in your handle :)

If you're going to add a comment saying it should go, it should also explain why and why it's not gone yet.

Reply:
Ack.